### ISSUE TITLE
AnimationEndListener sometimes becomes null and onAnimationEnd is never called

### ISSUE THREAD

**Issue by Kolyall** [florent37/ExpectAnim#25]
AnimationEndListener sometimes becomes null in your lib at the line:
```
final AnimationEndListener listener = endListenerWeakReference.get();
```
For example
```
new ExpectAnim()
                .expect(mView)
                .toBe(
                       .atItsOriginalPosition()
                ).toAnimation()
                .setDuration(1000)
                .setEndListener(new AnimationEndListener() {
                    @Override
                    public void onAnimationEnd(ExpectAnim expectAnim) {
//NB: sometimes AnimationEndListener is null in your lib and the method doesn't work
                        //do smth
                    }
                }).start();
```

**Comment by Pirokar** [comment]
confirm, when it will be fixed? Very interesting library, but this bug negates all...

**Comment by florent37** [comment]
just fixed, can you try with 1.0.4 ? 👍
